TotalCrossSDK/build.gradle

```diff
@@ -37,7 +37,7 @@ retrolambda {
 }
 
 wrapper {
-    gradleVersion = '5.0'
+    gradleVersion = '5.6.4'
```

Comment:
It can generate conflicts because it is an "unrelated change" (i know, are related but I understand that not in scope, it is a change to make it work). We even have a PR that goes up the version of Gradle. See #199.